webapp/src/components/github_label_selector/github_label_selector.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+import IssueAttributeSelector from 'components/issue_attribute_selector';
+
+export default class GithubLabelSelector extends PureComponent {
+    static propTypes = {
+        repo: PropTypes.string,
+        theme: PropTypes.object.isRequired,
+        labels: PropTypes.array.isRequired,
+        onChange: PropTypes.func,
+        actions: PropTypes.shape({
+            getLabels: PropTypes.func.isRequired,
+        }).isRequired,
+    };
+
+    handleChange = (items) => {
+        if (!items || items.length === 0) {
+            return;
+        }
+
+        this.props.onChange(items);
+    };
+
+    render() {
+        const loadLabels = async () => {
```

Comment:
This can be a method on the component, rather than a function declared inside of `render`?